cli/src/action/database/sqlite.rs

```diff
@@ -33,6 +33,16 @@ const DEFAULT_SQLITE: &str = "splinter_state.db";
 
 /// Run the sqlite migrations against the provided connection string
 pub fn sqlite_migrations(connection_string: String) -> Result<(), CliError> {
+    if let Err(err) = OpenOptions::new()
```

Comment:
Should exclude the special connection string ":memory:" from this check.

Reply:
It is now excluded